osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -68,24 +76,35 @@ func addPackageInstallTest(pkgTestSetup []*packageManagementTestSetup) []*packag
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
 		var assign *osconfigpb.Assignment
-		var instaneName, ss, vs string
+		var instanceName, image, ss, vs string
 
 		switch pkgManager {
 		case "apt":
-			instaneName = fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName)
+			image = debianImage
+			instanceName = fmt.Sprintf("%s-%s", filepath.Base(image), testName)
```

Comment:
path.Base, also this is better to have down in line 107 as its the same for all pkgManagers

Reply:
same